rollup.config.mjs

```diff
@@ -232,7 +232,8 @@ function buildTarget(buildType, moduleFormat) {
             values: {
                 ...sdkVersion,
                 _DEBUG: 1,
-                _PROFILER: 1
+                _PROFILER: 1,
+                _ASSET_BASE_URL: JSON.stringify(process.env.ASSET_BASE_URL) || false
```

Comment:
When developing locally and using `use_local_engine` it is necessary to set `process.env.ASSET_BASE_URL=https://launch.local-playcanvas.com` as scripts are imported relative to the engine, which would be pointing to localhost